docs/home/DEPRECATION-NOTE.md

```diff
@@ -1,40 +1,79 @@
+removal in Epiphany version 2.0.
 
 The list of components removed:
 
 - Hashicorp Vault
 
-The role related to Hashicorp Vault has never been developed beyond MVP/PoC stage and cannot be used for production usage. It also causes troubles with upgrade to new Kubernetes versions.
+The role related to Hashicorp Vault has never been developed beyond MVP/PoC stage and cannot be used for production
+usage. It also causes troubles with upgrade to new Kubernetes versions.
 
-Removed: Epiphany 2.0
-Issue: [2834](https://github.com/epiphany-platform/epiphany/issues/2834)
+Removed: Epiphany 2.0 Issue: [2834](https://github.com/epiphany-platform/epiphany/issues/2834)
 
 - Istio
 
-We are considering replacement of Istio component with different solution. Additionally, Istio causes problems during upgrade of Kubernetes.
+We are considering replacement of Istio component with different solution. Additionally, Istio causes problems during
+upgrade of Kubernetes.
 
-Removed: Epiphany 2.0
-Issue: [2836](https://github.com/epiphany-platform/epiphany/issues/2836)
+Removed: Epiphany 2.0 Issue: [2836](https://github.com/epiphany-platform/epiphany/issues/2836)
 
 - Apache Ignite
 
 Apache Ignite is prone to 1.x log4j issue.
 
-Removed: Epiphany 2.0
-Issue: [2837](https://github.com/epiphany-platform/epiphany/issues/2837)
+Removed: Epiphany 2.0 Issue: [2837](https://github.com/epiphany-platform/epiphany/issues/2837)
 
 - Logstash
 
-The functionality that the Logstash is serving in Epiphany - exporting csv can right now be achieved using the Open Distro for Elasticsearch plugin. The Logstash in this version is prone to log4j issue and this is also the reason why should be removed.
+The functionality that the Logstash is serving in Epiphany - exporting csv can right now be achieved using the Open
+Distro for Elasticsearch plugin. The Logstash in this version is prone to log4j issue and this is also the reason it
+should be removed.
 
-Removed: Epiphany 2.0
-Issue: [2833](https://github.com/epiphany-platform/epiphany/issues/2833)
+Removed: Epiphany 2.0 Issue: [2833](https://github.com/epiphany-platform/epiphany/issues/2833)
 
 ## Next steps
 
-If you plan Epiphany upgrade from v1.x to v2.x and Logstash is installed in your cluster but you don't use it, it is recommended to:
+### Logstash
+
+If you plan Epiphany upgrade from v1.x to v2.x and Logstash is installed in your cluster, but you don't use it, it is
+recommended to:
+
 - remove `logstash` group, if present, from `inventory` file
 - remove Logstash from machines (recommended way is to use `apt`/`yum` according to your OS)
+
+### Ignite
+
+There are a few installation approaches that influence the removal of Ignite.
+
+#### Standalone installation on VMs
+
+To remove Ignite application in this case, it's necessary to destroy related infrastructure part with following
+configuration. To be sure about compatibility of applied changes, use the same epicli version.
```

Comment:
Maybe worth mentioning that `epicli apply` with `--skip-config` may be used to run only terraform part and skip ansible role provisioning.